demos/whisper-base/whisper.js

```diff
@@ -101,6 +102,19 @@ export class Whisper {
                     deviceType: this.deviceType,
                 },
             ],
+            preferredOutputLocation: this.iobinding ? (() => {
+                const pairs = {};
```

Comment:
```javascript
                let pairs = {};
```
It feels weird to use const here even though you intend to modify it.

Reply:
Done.